spec/acceptance/activesupport_instrumentation_spec.rb

```diff
@@ -97,13 +97,15 @@ def subscribed(callback, *args)
   end
 
   it "tracks proper time of compiling the factory" do
-    time_to_execute = 0
-    callback = ->(_name, start, finish, _id, _payload) { time_to_execute = finish - start }
+    time_to_execute = {user: 0}
+    callback = ->(_name, start, finish, _id, _payload) {
+      time_to_execute[_payload[:name]] = (finish - start)
```

Comment:
A minimal change here: since `payload` is now used, it shouldn't be preceded with an underscore:
```suggestion
    callback = ->(_name, start, finish, _id, payload) {
      time_to_execute[payload[:name]] = (finish - start)
```